sale_order_product_recommendation_secondary_unit/README.rst

```diff
@@ -14,13 +14,13 @@ Sale Order Product Recommendation Secondary Unit
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
-    :target: https://github.com/OCA/sale-workflow/tree/13.0/sale_order_product_recommendation_secondary_unit
+    :target: https://github.com/OCA/sale-workflow/tree/14-mig-sale-order-product-recommendation-secondary-unit/sale_order_product_recommendation_secondary_unit
```

Comment:
@tcthai could you pls check the readme ? it's not correct, you can ref this [one](https://github.com/OCA/sale-workflow/pull/1778/commits/3b4f1c05e2b8b41cc2e195a481b985141ad88475#diff-9f75438fdf6eebe7bf7015b27be3a1aaf707a680b54c9d58d795dd3b9652fd0e)